omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -90,6 +90,12 @@
 			<spring:nestedPath path="user.person.names[0]">
 				<openmrs:portlet url="nameLayout" id="namePortlet" size="full" parameters="layoutMode=edit|layoutShowTable=false|layoutShowExtended=false" />
 			</spring:nestedPath>
+			<c:if test="${hasEmailField}">
+				<tr>
+					<td><openmrs:message code="Email" /><span class="required">*</span></td>
```

Comment:
Is the email field required?

Reply:
It is literally not required.  However, since reseting password actually depends on it, i thought it would be a good practice if we emphasise users to always fill it

Reply:
Required means we cannot save the user without it.

Reply:
> reseting password

I'm not sure I necessarily follow this. We don't (in the general case) have support for sending user's email for password resets, though it's possible for a module to add this capability.

Reply:
@ibacher I am simply trying to make sure that [rest/v1/passwordreset](https://github.com/openmrs/openmrs-module-webservices.rest/blob/417b2775e1f4c0a11cf564bb3d7c5778337181b7/omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/PasswordResetController2_2.java#L32) can get working so that we can use it in the frontend  
This PR is might be followed by many more until this works